store/queries/candidate_actors.sql

```diff
@@ -25,9 +25,22 @@ WHERE
     did = sqlc.arg(did)
 RETURNING *;
 
+-- name: UpdateActorProfile :exec
+WITH ap as (
+    INSERT INTO actor_profiles
+        (actor_did, id, created_at, indexed_at, display_name, description)
+    VALUES
+        (sqlc.arg(did), sqlc.arg(id), sqlc.arg(updated_at), sqlc.arg(indexed_at), sqlc.arg(display_name), sqlc.arg(description))
+    RETURNING actor_did, id
+)
+UPDATE candidate_actors ca
+SET current_profile_id = (SELECT id FROM ap)
+WHERE
+    did = (SELECT actor_did FROM ap);
```

Comment:
I'm curious what the perf would be like if rather than maintaining a `current_profile_id`, we just did a join that fetched the most recent actor_profile instead (I'm aware this is likely to be quite awkward - and dubious that that join would perform well...)

I suppose there's a potential risk this could end up being racey in its current form.

Reply:
On second though, I think what you are doing is likely to be way more performant, even at the risk of raciness - let's stick with this.